back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/idea_plaintext_parser_service.rb

```diff
@@ -9,10 +9,20 @@ class IdeaPlaintextParserService
     EMPTY_SELECT_CIRCLES = ['O', '○']
     EMPTY_MULTISELECT_SQUARES = ['☐']
 
-    def initialize(custom_fields, locale)
+    def initialize(participation_context, custom_fields, locale)
       @custom_fields = custom_fields
       @locale = locale
 
+      @form_title = get_form_title participation_context
+
+      pdf = PrintCustomFieldsService.new(
+        participation_context,
+        custom_fields,
+        { locale: locale }
+      ).create_pdf
+
+      @number_of_pages_per_form = pdf.page_count
```

Comment:
Would be more efficient to just pass in `@pdf_form_page_count` which the calling class is already calculating and using in a couple of places